Packs/ipinfo/ReleaseNotes/2_1_3.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### New: ipinfo (Deprecated)
+- Restore ipinfo integration
```

Comment:
I don't like the fact we're calling it "New". Let's remove it - validate should fail in the branch, but we can force merge it.
```suggestion
#### Integrations
##### ipinfo (Deprecated)
- Restore **ipinfo** integration.
```